debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/commands/J9ConstantPoolCommand.java

```diff
@@ -0,0 +1,539 @@
+import com.ibm.j9ddr.vm29.j9.walkers.ClassSegmentIterator;
+import com.ibm.j9ddr.vm29.pointer.I64Pointer;
+import com.ibm.j9ddr.vm29.pointer.PointerPointer;
+import com.ibm.j9ddr.vm29.pointer.SelfRelativePointer;
+import com.ibm.j9ddr.vm29.pointer.U16Pointer;
+import com.ibm.j9ddr.vm29.pointer.U32Pointer;
+import com.ibm.j9ddr.vm29.pointer.generated.J9BuildFlags;
+import com.ibm.j9ddr.vm29.pointer.generated.J9ClassPointer;
+import com.ibm.j9ddr.vm29.pointer.generated.J9ConstantPoolPointer;
+import com.ibm.j9ddr.vm29.pointer.generated.J9InitializerMethodsPointer;
+import com.ibm.j9ddr.vm29.pointer.generated.J9JavaVMPointer;
+import com.ibm.j9ddr.vm29.pointer.generated.J9MethodPointer;
+import com.ibm.j9ddr.vm29.pointer.generated.J9ObjectPointer;
+import com.ibm.j9ddr.vm29.pointer.generated.J9RAMClassRefPointer;
+import com.ibm.j9ddr.vm29.pointer.generated.J9RAMConstantDynamicRefPointer;
+import com.ibm.j9ddr.vm29.pointer.generated.J9RAMConstantPoolItemPointer;
+import com.ibm.j9ddr.vm29.pointer.generated.J9RAMFieldRefPointer;
+import com.ibm.j9ddr.vm29.pointer.generated.J9RAMInterfaceMethodRefPointer;
+import com.ibm.j9ddr.vm29.pointer.generated.J9RAMMethodHandleRefPointer;
+import com.ibm.j9ddr.vm29.pointer.generated.J9RAMMethodRefPointer;
+import com.ibm.j9ddr.vm29.pointer.generated.J9RAMMethodTypeRefPointer;
+import com.ibm.j9ddr.vm29.pointer.generated.J9RAMStaticFieldRefPointer;
+import com.ibm.j9ddr.vm29.pointer.generated.J9RAMStringRefPointer;
+import com.ibm.j9ddr.vm29.pointer.generated.J9ROMClassPointer;
+import com.ibm.j9ddr.vm29.pointer.generated.J9ROMClassRefPointer;
+import com.ibm.j9ddr.vm29.pointer.generated.J9ROMConstantDynamicRefPointer;
+import com.ibm.j9ddr.vm29.pointer.generated.J9ROMConstantPoolItemPointer;
+import com.ibm.j9ddr.vm29.pointer.generated.J9ROMFieldRefPointer;
+import com.ibm.j9ddr.vm29.pointer.generated.J9ROMMethodHandleRefPointer;
+import com.ibm.j9ddr.vm29.pointer.generated.J9ROMMethodPointer;
+import com.ibm.j9ddr.vm29.pointer.generated.J9ROMMethodRefPointer;
+import com.ibm.j9ddr.vm29.pointer.generated.J9ROMMethodTypeRefPointer;
+import com.ibm.j9ddr.vm29.pointer.generated.J9ROMNameAndSignaturePointer;
+import com.ibm.j9ddr.vm29.pointer.generated.J9ROMSingleSlotConstantRefPointer;
+import com.ibm.j9ddr.vm29.pointer.generated.J9ROMStringRefPointer;
+import com.ibm.j9ddr.vm29.pointer.helper.J9ClassHelper;
+import com.ibm.j9ddr.vm29.pointer.helper.J9ObjectHelper;
+import com.ibm.j9ddr.vm29.pointer.helper.J9RASHelper;
+import com.ibm.j9ddr.vm29.pointer.helper.J9ROMClassHelper;
+import com.ibm.j9ddr.vm29.pointer.helper.J9UTF8Helper;
+import com.ibm.j9ddr.vm29.structure.J9Class;
+import com.ibm.j9ddr.vm29.structure.J9ConstantPool;
+import com.ibm.j9ddr.vm29.structure.J9VTableHeader;
+import com.ibm.j9ddr.vm29.types.UDATA;
+
+public class J9ConstantPoolCommand extends Command {
+	private static final long J9VTABLE_INITIAL_VIRTUAL_OFFSET = J9Class.SIZEOF + J9VTableHeader._initialVirtualMethodOffset_;
```

Comment:
The type `J9VTableHeader` has not always existed: this reference may trigger a `NoClassDefFoundError`.